src/components/Notifications/Category.tsx

```diff
@@ -0,0 +1,103 @@
+import { FC } from "react"
+import { Pressable, StyleSheet, View } from "react-native"
+import { Text } from "components/Text"
+import { Switch } from "react-native-switch"
+// import { IconProps } from "assets/settings"
+import { usePalette } from "utils/colors"
+
+export interface CategoryProps {
+  // icon?: IconProps
+  title: string
+  switchControl?: {
+    /** State of the switch */
+    toggled: boolean
+    /** Function fired when the state of the switch changes */
+    onToggle: (value: boolean) => void
+  }
+  notifications?: number
+  onClick?: () => void
+}
+
+export const Category: FC<CategoryProps> = props => {
+  const { backgroundSecondary, palette, isLight } = usePalette()
+
+  const notifications = props.notifications ?? 0
+
+  return (
+    <Pressable style={styles.container} onPress={props.onClick}>
+      {notifications ? (
+        <View style={styles.circle}>
+          <Text>{props.notifications}</Text>
+        </View>
+      ) : undefined}
+      <Text>{props.title}</Text>
+
+      <View style={styles.switch}>
+        <Switch
+          value={props.switchControl?.toggled}
+          onValueChange={value => {
+            props.switchControl?.onToggle(value)
+          }}
+          changeValueImmediately={true}
+          renderActiveText={false}
+          renderInActiveText={false}
+          barHeight={27}
+          switchWidthMultiplier={3}
+          circleSize={18}
+          circleActiveColor={backgroundSecondary}
+          circleInActiveColor={palette.accent}
+          circleBorderWidth={0}
+          innerCircleStyle={{
+            borderWidth: 1,
+            borderColor: !props.switchControl?.toggled
+              ? palette.accent
+              : isLight
+              ? "#EBEBEB"
+              : "#3A4257",
+          }}
+          backgroundActive={palette.accent}
+          backgroundInactive={"#FFF"}
+          containerStyle={{
+            borderWidth: 1,
+            borderColor: palette.accent,
+          }}
+          switchLeftPx={1.5}
+          switchRightPx={1.3}
+        />
+      </View>
+    </Pressable>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    width: "90%",
+    top: "24%",
+    marginTop: "8%",
```

Comment:
non mi piacciono troppo sti valori in percentuale, e `top` non serve dentro a questo componente, al massimo sarà il parent ad aggiungere un padding, ad occhio e croce direi che `marginTop` può essere 35 come la distanza che c'è su Figma e al posto di una `width` dai un `marginHorizontal` in pixel, credo tipo 18 dovrebbe andare bene, controlla tu bene tutti i valori